Source/WebKit/UIProcess/win/WebProcessPoolWin.cpp

```diff
@@ -86,4 +89,14 @@ void WebProcessPool::platformResolvePathsForSandboxExtensions()
 {
 }
 
+#if ENABLE(REMOTE_INSPECTOR)
+void WebProcessPool::setPagesControlledByAutomation(bool controlled)
```

Comment:
Do you have a plan to use this method for PlayStation port too?
If so, why not adding this to the common WebProcessPool.cpp with `#if PLATFORM(WIN)`?

Reply:
Yes, we will use this method for playstation port.
I'll move this to the common WebProcessPool.cpp.